app/clients/aws.js

```diff
@@ -4,12 +4,18 @@ import localStore from '../store/localStore';
 
 process.env.AWS_SDK_LOAD_CONFIG = true;
 
-const credentials = new AWS.SharedIniFileCredentials({
-  profile: localStore.get('profile')
+const profile = localStore.get('profile');
+const sharedIniFileCredentials = new AWS.SharedIniFileCredentials({ profile });
+const processCredentials = new AWS.ProcessCredentials({ profile });
```

Comment:
For now I've only elected to add `ProcessCredentials` as that's my required use case, but other credential providers could be added to this chain to fully replicate the behavior of the AWS CLI. I figured there would be a default credential provider chain in the AWS SDK that I could use, but I didn't see one...

Reply:
No worries; I think this makes sense. Let's add more providers as we uncover more usecases!